test-specifications/utility.md

```diff
@@ -0,0 +1,293 @@
+# Utility JSON RPC Methods
+
+## Description
+The JSON RPC methods mentioned in this file describe additional methods that should be implemented by a TCK server that provide a utility value of some sort that is not specific to one Hedera request type. These methods can involve, but are not limited to, setting up or tearing down a test environment or using the SDK to generate a key pair to be used by the TCK driver.
+
+## Methods
+
+### `setup`
```

Comment:
There was a "setup1" example in the previous version which connect to the testnet, do we still have it?

Reply:
Got it. I think it makes sense to add this info to the updated doc then.

Reply:
Updated with the setup information and assumptions